src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -23,9 +24,12 @@ public class CounterMonitor
         private List<string> _counterList;
         private CancellationToken _ct;
         private IConsole _console;
+        private ICounterExporter exporter;
```

Comment:
Nit: We should stick to a convention for field names. Leading underscore is typical in BCL code and appears to be somewhat adopted here, so I would go with that.